tables/ru-litbrl.ctb

```diff
@@ -7,14 +7,14 @@
 #-index-name: Russian
 #-display-name: Russian braille
 #
-#+language: ru
+#+locale: ru
```

Comment:
"locale" was deprecated in v3.21. See https://github.com/liblouis/liblouis/wiki/Table-discovery-based-on-table-metadata.